configs/sample_params/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
```

Comment:
Do you think this crate should be here? 
Before `configs` wasn't used to store source code. Maybe use `config` folder?

Reply:
Or just root folder.

Reply:
<kbd>/configs</kbd> folder is not for crates for sure. I'd propose using <kbd>/sample_params</kbd> for now, and later unify with samples removed from `iroha` crate (https://github.com/hyperledger/iroha/issues/4136)

Reply:
Ok, it makes sense to rename the crate to more specific one since it is currently only used for aliases

Reply:
Maybe not use `iroha_` prefix for it?

We have `test_network` and not `iroha_test_network`. And eventually `iroha::samples` might become `test_samples`. You can make `test_samples` already now and place your code there.

Reply:
we will remove `test_network`